tests/Detector.Tests/Hugo/HugoDetectorTest.cs

```diff
@@ -22,8 +23,8 @@ public HugoDetectorTest(TestTempDirTestFixture testFixture)
         }
 
         [Theory]
-        [InlineData(HugoConstants.TomlFileName)]
-        [InlineData(HugoConstants.ConfigFolderName, HugoConstants.TomlFileName)]
+        [InlineData("config.toml")]
+        [InlineData(HugoConstants.ConfigFolderName, "config.toml")]
         public void IsHugoApp_ReturnsTrue_ForAppWithConfigTomlFile(params string[] subPaths)
```

Comment:
Let's introduce the same set of tests that we have in this file but for the `hugo.*` files that we now support to ensure that we're detecting them properly.